scripts/bootstrap.sh.in

```diff
@@ -182,24 +254,31 @@ ${gpg_keys:+gpgkey=${gpg_keys%?}}
 EOF
 }
 
-install_salt_minion() {
-    local -a yum_opts=(
-        '--assumeyes'
-        '--disablerepo=*'
-        '--enablerepo=metalk8s-*'
-    )
-
-    "$YUM" install "${yum_opts[@]}" salt-minion
+configure_apt_local_repository() {
+    local -r repo_name=$1
+    local -r repo_path="$BASE_DIR/packages/debian/$repo_name"
+    echo "deb [trusted=yes] file://$repo_path bionic $repo_name" \
+        > /etc/apt/sources.list.d/"$repo_name".list
 }
 
-install_genisoimage() {
-    local -a yum_opts=(
-        '--assumeyes'
-        '--disablerepo=*'
-        '--enablerepo=metalk8s-*'
-    )
-
-    "$YUM" install "${yum_opts[@]}" genisoimage
+install_packages() {
+    local -ra packages=("$@")
+
+    case "$OS" in
+        centos)
+            local -a yum_opts=(
+                '--assumeyes'
+                '--disablerepo=*'
+                '--enablerepo=metalk8s-*'
+            )
+            "$YUM" install "${yum_opts[@]}" "${packages[@]}"
+            ;;
+        ubuntu)
+            DEBIAN_FRONTEND=noninteractive "$APT" -y install "${packages[@]}"
```

Comment:
Not sure if we want to do it also on Ubuntu but, on CentOS we disable all repositories that are not `metalk8s-*` when installing the packages.
I know there is no clean way to do it, but we can do something like:
```
local -a repo_list=(/etc/apt/sources.list)

map_file -O 1 -t repo_list < <(find /etc/apt/sources.list.d/ ! -name 'metalk8s-*' -a -name '*.list')

for source_list in "${repo_list[@]}"; do
    mv "$source_list"{,.disabled}
done

## Install your packets
for source_list in "${repo_list[@]}"; do
    mv "$source_list"{.disabled,}
done
```

Reply:
Done, thank you ! I done some tests on Kubernetes and it was well downloaded from the new internal repo list, but it's more safe like this !

Reply:
Sorry, one last thing I should have told you.
You should prevent the script from failing here, otherwise you could end up with system repositories in a bad state (it should not, but hey better be safe than sorry).